bfh/__init__.py

```diff
@@ -130,7 +130,10 @@ def _raw(self):
         out = dict(**self._raw_input)
         for name in self._field_names:
             value = getattr(self, name)
-            out[name] = value
+            if isinstance(value, Schema):
```

Comment:
We should also cover the situation where a Schema is nested within a GenericSchema. So let's use hasattr here or refactor `_raw` onto SchemaInterface and check for that, and make sure GenericSchema._raw does similar. The end goal is a a tree of any mix of Schema and GenericSchema should be transformed into a tree of all GenericSchema.